bigbluebutton-html5/imports/api/meetings/server/publishers.js

```diff
@@ -15,8 +15,10 @@ function meetings(credentials) {
   const selector = {
     $or: [
       { meetingId },
-      { 'meetingProp.isBreakout': true },
-      { 'breakoutProps.parentId': meetingId },
+      {
+        'meetingProp.isBreakout': true,
+        'breakoutProps.parentId': meetingId,
```

Comment:
:+1:  tested - this change works

Reply:
(tricky)We may need one more condition here. If a viewer is subscribing, ideally that user should not be able to see the other breakouts that he's not allowed to enter. The tricky part is due to the fact that we may have a 'allowed to select room to join' check where the viewer would be allowed to see this info.